src/App.js

```diff
@@ -10,10 +11,32 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends Component {
+  state = {
+    selectedTab: tabs[0],
+  }
+
+  onTabSelected = (event, tabID) => {
+    event.preventDefault();
```

Comment:
not sure, do you need this? can you explain for what

Reply:
I use onTabSelected on <a> tag. I don't want to go where src link leads me;)